contracts/utils/TokenPriceHelper.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.8.10;
+
+import "../DS/DSMath.sol";
+import "../utils/TokenUtils.sol";
+import "../interfaces/lido/IWStEth.sol";
+import "./helpers/UtilHelper.sol";
+import "../interfaces/chainlink/IFeedRegistry.sol";
+import "./Denominations.sol";
+
+contract TokenPriceHelper is DSMath, UtilHelper {
+
+    IFeedRegistry public constant feedRegistry = IFeedRegistry(CHAINLINK_FEED_REGISTRY);
+
+    function getRoundInfo(address _inputTokenAddr, uint80 _roundId) public view returns (uint256, uint256 updateTimestamp) {
+        address tokenAddr = _inputTokenAddr;
+
+        if (_inputTokenAddr == TokenUtils.WETH_ADDR) {
```

Comment:
Handle here btc as well as in getRoundInfo(), maybe we can extract the 3 ifs and wsteth handling after as an modifier that both function implement.